### ISSUE TITLE
Things to be done

### ISSUE THREAD

**Issue by Felix-Zhenghao** [Felix-Zhenghao/VideoPlan#2]
- Adopt vlm loss (text autoregrassive loss) & loss weighting
- Adopt flow matching
- Impl training time evaluation
- Enable pytorch profile so can accordingly optimize the training code
